packages/react-dom/src/events/DOMModernPluginEventSystem.js

```diff
@@ -165,6 +168,43 @@ export function listenToEvent(
   }
 }
 
+const validFBLegacyPrimerRels = new Set([
+  'dialog',
+  'dialog-post',
+  'async',
+  'async-post',
+  'theater',
+  'toggle',
+]);
+
+function willDeferLaterForFBLegacyPrimer(nativeEvent: any): boolean {
+  let node = nativeEvent.target;
+  const type = nativeEvent.type;
+  if (type !== 'click') {
+    return false;
+  }
+  while (node !== null) {
+    // Primer works by intercepting a click event on an <a> element
+    // that has a "rel" attribute that matches one of the valid ones
+    // in the Set above. If we intercept this before Primer does, we
+    // will need to defer the current event till later and discontinue
+    // execution of the current event. To do this we can add a document
+    // event listener and continue again later after propagation.
+    if (node.tagName === 'A' && validFBLegacyPrimerRels.has(node.rel)) {
+      const legacyFBSupport = true;
+      trapEventForPluginEventSystem(
+        document,
+        ((type: any): DOMTopLevelEventType),
+        false,
+        legacyFBSupport,
```

Comment:
What am I missing about the `legacyFBSupport ` param? `trapEventForPluginEventSystem` only accepts 3 params. Does FB override this somewhere I'm not seeing? Is this a mistake?

Reply:
This PR makes `trapEventForPluginEventSystem` take 4 arguments. If you look through the PR you should see it being added. Sometimes the github whitespace makes this harder to see, sorry :/

Reply:
Weird. I searched the entire files page and only found the import and declaration on this line. Huh!

Reply:
Ooh that's because the name was truncated from the diff view because of multi-line param formatting. Oops!

Reply:
Even now? That's some serious caching issue on GitHub's part. I frequently have to do Cmd+Shift+R to make Github update PRs these days though, so it's unsurprising.